File: C Modules/Assignments/Assignment no 7/Question-2/Assignment no 3/Type-3/Question_9.c
#include<stdio.h>
int palindrome()
{
    int num,temp,rev=0,digit;
    int *ptr1=&num;
    printf("Enter the number:");
    scanf(" %d", ptr1);
    temp=*ptr1;
    while (temp>0)
    {
        digit=temp%10;
        rev=rev*10+digit;
        temp=temp/10;
    }
    if (rev==*ptr1)
    {
            return 1;
    }
    else{
            return 0;
    }
}
int main()
{
    int result=palindrome();
    int *ptr2=&result;
    if (*ptr2==1)
    {
        printf(" is a palindrome");
    }
    else{
        printf(" is not a palindrome");
    }
return  0;
}

File: Java Module/Assignments in c to java/Assignment-03/Question_9.java
public class Question_9 {

    public static void main(String[] args) {
            int num=121,temp,rev=0,digit;
    // printf("Enter the number:");
    // scanf(" %d", &num);
    temp=num;
    while (temp>0)
    {
        digit=temp%10;
        rev=rev*10+digit;
        temp=temp/10;
    }
    if (rev==num)
    {
        System.out.printf("%d is a palindrome",num);
    }
    else{
        System.out.printf("%d is not a palindrome",num);
    }
    }
}